### ISSUE TITLE
[documentation][python] GradingPrimary: missing expected value type doc

### ISSUE THREAD

**Issue by MrLixm** [AcademySoftwareFoundation/OpenColorIO#1640]
Hello,
When looking at the GradingPrimary class documentation (https://opencolorio.readthedocs.io/en/latest/api/grading_transforms.html),
we can see a bunch of property is available but none of them specifies which type of value is expected when **set**.

This would have not been an issue if they all except the same type but seems some except a `GradingRGBM` instance while others just expect a `float`.

Would be cool to have in the documentation what type is expected.

Here are the tests I ran to check who accepts what

```python
import unittest

import PyOpenColorIO as ocio


class OcioGradingPrimary(unittest.TestCase):
    @staticmethod
    def to_rgbm(v):
        rgbm = ocio.GradingRGBM()
        rgbm.master = v
        return rgbm

    def test_properties(self):

        gp = ocio.GradingPrimary(ocio.GRADING_LIN)

        gp.brightness = self.to_rgbm(0.18)
        gp.clampBlack = 0.18
        gp.clampWhite = 0.18
        gp.contrast = self.to_rgbm(0.18)
        gp.exposure = self.to_rgbm(0.18)
        gp.gain = self.to_rgbm(0.18)
        gp.gamma = self.to_rgbm(0.18)
        gp.lift = self.to_rgbm(0.18)
        gp.offset = self.to_rgbm(0.18)
        gp.pivot = 0.18
        gp.saturation = 0.18

        gp.brightness = self.to_rgbm(0.18)
        with self.assertRaises(TypeError): gp.clampBlack = self.to_rgbm(0.18)
        with self.assertRaises(TypeError): gp.clampWhite = self.to_rgbm(0.18)
        gp.contrast = self.to_rgbm(0.18)
        gp.exposure = self.to_rgbm(0.18)
        gp.gain = self.to_rgbm(0.18)
        gp.gamma = self.to_rgbm(0.18)
        gp.lift = self.to_rgbm(0.18)
        gp.offset = self.to_rgbm(0.18)
        with self.assertRaises(TypeError): gp.pivot = self.to_rgbm(0.18)
        with self.assertRaises(TypeError): gp.saturation = self.to_rgbm(0.18)


if __name__ == "__main__":
    unittest.main()
```

Cheers.
Liam.

**Comment by sumana-2705** [comment]
@MrLixm  can you help me where can I get to know more about this documentation to gain knowledge in type of expected value? I want to work on this

**Comment by MrLixm** [comment]
Hello, sorry but I am not familiar with how the OCIO documentation is built.
I am only an user for it and as mentioned in the first message the python documentation doesn't mention which data type must be used:
![image](https://github.com/AcademySoftwareFoundation/OpenColorIO/assets/64362465/7d1687f2-82de-42f7-ab77-048c8b8084d9)
While the C++ documentation indicate the type:
![image](https://github.com/AcademySoftwareFoundation/OpenColorIO/assets/64362465/9a0a2d84-fc86-4670-a2c8-77040a5c447b)
https://opencolorio.readthedocs.io/en/latest/api/grading_transforms.html#gradingprimary

Hope this helps,
Liam.

**Comment by sumana-2705** [comment]
yeah it works, thanks for information, should I need to perform additional tests or update the results based on these test results, which was provided by you, that is which property accepts which type? @MrLixm

**Comment by sumana-2705** [comment]
I have opened a pull request, please review it suggest me if any changes required. 
Thank you

**Comment by doug-walker** [comment]
Closing this as fixed in PR #1989.
